ccd-definition/RoleToAccessProfiles/RoleToAccessProfiles-CUI-nonprod.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "RoleName" : "Claimant",
+    "Authorisation": "",
+    "CaseTypeID": "CIVIL",
+    "AccessProfiles": "CLAIMANT",
+    "ReadOnly": "N",
+    "Disabled": "F",
+    "CaseAccessCategories": ""
+  },
+  {
+    "RoleName" : "Defendant",
+    "Authorisation": "",
+    "CaseTypeID": "CIVIL",
+    "AccessProfiles": "DEFENDANT",
+    "ReadOnly": "N",
+    "Disabled": "F",
+    "CaseAccessCategories": ""
```

Comment:
should not these have access to spec claim: `    "CaseAccessCategories": "SPEC_CLAIM"`